### ISSUE TITLE
Format button is disabled when go to a document from the drop down arrow

### ISSUE THREAD

**Issue by JieCarolHu** [dotnet/roslyn#26048]
**Version Used**: 
15.7.0 preview 3 (27604)

**Steps to Reproduce**:

1.  Open 10 nuspec files together
2.  Select a file from the drop down arrow 
![image](https://user-images.githubusercontent.com/17056651/38526972-56fb9170-3c0d-11e8-871a-4f8f06ab6924.png)
3. The file open but the Format button is disabled

**Expected Behavior**:

**Actual Behavior**:

**Comment by CyrusNajmabadi** [comment]
This does not seem roslyn related.  @olegtk Does this belong to editor somehow?

**Comment by olegtk** [comment]
yeah, sounds like an editor bug. @jinujoseph please route to the editor. Thanks!

**Comment by CyrusNajmabadi** [comment]
Closing as external.
